src/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from "react";
+import { useTheme } from "@mui/material/styles";
+import Box from "@mui/material/Box";
+import MobileStepper from "@mui/material/MobileStepper";
+import Paper from "@mui/material/Paper";
+import Typography from "@mui/material/Typography";
+import Button from "@mui/material/Button";
+import KeyboardArrowLeft from "@mui/icons-material/KeyboardArrowLeft";
+import KeyboardArrowRight from "@mui/icons-material/KeyboardArrowRight";
+import SwipeableViews from "react-swipeable-views";
+import { autoPlay } from "react-swipeable-views-utils";
+import { SLIDER_IMAGES } from "../../common/constants";
+
+const AutoPlaySwipeableViews = autoPlay(SwipeableViews);
+
+function Slider() {
+  const theme = useTheme();
+  const [activeStep, setActiveStep] = React.useState(0);
+  const maxSteps = SLIDER_IMAGES.length;
+
+  const handleNext = () => {
+    setActiveStep((prevActiveStep) => prevActiveStep + 1);
+  };
+
+  const handleBack = () => {
+    setActiveStep((prevActiveStep) => prevActiveStep - 1);
+  };
+
+  const handleStepChange = (step: React.SetStateAction<number>) => {
+    setActiveStep(step);
+  };
+
+  return (
+    <Box display="flex" justifyContent="center" alignItems="center">
+      <Box sx={{ maxWidth: 1066 }}>
+        <AutoPlaySwipeableViews
+          axis={theme.direction === "rtl" ? "x-reverse" : "x"}
+          index={activeStep}
+          onChangeIndex={handleStepChange}
+          enableMouseEvents
+        >
+          {SLIDER_IMAGES.map((step, index) => (
+            <div data-testid="swiper-div-img" key={step.label}>
+              {Math.abs(activeStep - index) <= 2 ? (
+                <Box
+                  component="img"
+                  sx={{
+                    height: 400,
```

Comment:
to improve the readability of the code, you can move the styles to a separate variable
Example:
```
const boxStyles = {
                    display: "block",
                    overflow: "hidden",
                    width: "100%",
                    borderRadius: "16px",
                  }
```

Then you can use it:
                      ` <Box sx={boxStyles} />`